### ISSUE TITLE
Can't run everything.sh

### ISSUE THREAD

**Issue by benjaminjellison** [panda-re/lava#12]
Trying to run and set it up as described in readme, I get this error when running ```everything.sh```:

```
[everything] PANDA taint analysis prospective bug mining -- input /bin/ls -- logging to /home/lava/lava/file_lava/logs/bug_mining--bin-ls.log
/usr/bin/python /home/lava/lava/scripts/bug_mining.py /home/lava/lava/file_lava.json /bin/ls
command failed! exit code was 1
========== end of logfile /home/lava/lava/file_lava/logs/bug_mining--bin-ls.log: ========== 

bug_mining.py /home/lava/lava/file_lava.json /bin/ls

[bug_mining.py] Entering /home/lava/lava.

Traceback (most recent call last):
  File "/home/lava/lava/scripts/bug_mining.py", line 128, in <module>
    command_args, installdir, isoname, project["expect_prompt"], rr=qemu_use_rr)
TypeError: create_recording() takes at least 8 arguments (8 given)

```
I can't find the create_recording definition to see what the problem is.

**Comment by moyix** [comment]
OK it was easier than I thought so I've committed what I hope is a fix (here: https://github.com/panda-re/lava/commit/1532f181248266ee5dc3f5e8ce83198174faf110). Could you check if it fixes your issue?

**Comment by moyix** [comment]
Hmm, it's possible further changes will be needed. I'll try to take a look soon. Sorry about this!

**Comment by benjaminjellison** [comment]
Had any chance to check this out yet?  I'm at SourceFest and was hoping to use LAVA (it only lasts 3 more days).  If not I can concentrate on something else, so if you could let me know one way or the other.

**Comment by moyix** [comment]
Yes, I finally got around to this today. I've pushed a couple of fixes and `everything.sh` now works out of the box on file again. Please give it a shot and let me know if they fix things for you!

**Comment by benjaminjellison** [comment]
It looks like it's working now.  The bug injection took almost 10 minutes, is that expected?  I only ask because the original timeout was 30 seconds, so I wanted to double check that minutes is a reasonable amount of time for the script to run.

**Comment by moyix** [comment]
The timeout in the script is just for running the command under PANDA; after that it does a taint analysis and then mines the taint log to construct bugs. It's expected that taint+bug mining will take around 10 minutes, yes; the amount of time is dependent on how big the program is (or more precisely, the length of the execution trace). So if you're trying to get a faster develop/test cycle, you could try using a smaller program too.

**Comment by AndrewFasano** [comment]
@benjaminjellison sounds like your issue was resolved, but if you're still having any problems, we just released a new version of LAVA and things should definitely work better now. We use a submodule of PANDA now so future changes there won't break LAVA again. Let us know if you have any more issues with it!
